crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -364,6 +365,56 @@ impl ConfigSettingAccessor<C8yRootCertPathSetting> for TEdgeConfig {
     }
 }
 
+impl ConfigSettingAccessor<MqttClientAddressSetting> for TEdgeConfig {
+    fn query(&self, _setting: MqttClientAddressSetting) -> ConfigSettingResult<String> {
+        Ok(self
+            .data
+            .mqtt
+            .client_address
+            .clone()
+            .unwrap_or("127.0.0.1".to_string()))
```

Comment:
```suggestion
            .unwrap_or(self.config_defaults.default_mqtt_bind_address.clone()))
```

Reply:
Please don't use the `self.config_defaults.default_mqtt_bind_address` setting here, as this would bring back coupling between the mqtt client settings and the binding settings (which we wanted to decouple in the first place). Have the a static reference to the loopback address (`127.0.0.1`) is fine here. Sure you can make it a constant somewhere, but just don't use the `bind` setting.

Reply:
Sure. I just wanted this value to be read from the central "config defaults" rather than it being an adhoc constant. And yes, the best thing to do would be to add a new `default_mqtt_client_address` in `TEdgeConfigDefaults`.